src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -256,6 +256,11 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				type: 'boolean',
 				description: localize('extensionsDeferredStartupFinishedActivation', "When enabled, extensions which declare the `onStartupFinished` activation event will be activated after a timeout."),
 				default: false
+			},
+			'extensions.experimental.quickAcessExtensions': {
```

Comment:
```suggestion
			'extensions.experimental.issueQuickAcess': {
```
how about this so it's a bit clearer. There's already an extensions quick access (for installing)